services/transactionService.ts

```diff
@@ -489,6 +500,24 @@ export async function fetchTransactionsByPaybuttonId (paybuttonId: string): Prom
   return transactions
 }
 
+export async function fetchTransactionsByPaybuttonIdGroupedByNetwork (
```

Comment:
This is created to be used in a place where the grouping is not relevant. I don't get why it exists, but in any case would be better if it took the txs list, not the button and networkids as a parameter.

Reply:
grouping is relevant, we order by date and group by network.. this way we separate the txs list.